weston-ivi-shell/src/ivi-share-gbm.c

```diff
@@ -33,8 +33,6 @@
 #include "ivi-share-server-protocol.h"
 #include "ivi-share.h"
 
-static uint32_t nativesurface_name;
```

Comment:
unrelated bug fixes. send in different PR.

Reply:
or at least reoder your patches that preparation patches come before feature patches and not in between